src/main/cpp/Robot.h

```diff
@@ -73,5 +74,6 @@ class Robot : public frc::TimedRobot
     AutonPreviewer *m_previewer;
     RobotState *m_robotState;
     SomeMech *m_someMech;
+    noteManagerGen *m_noteManagerGen;
```

Comment:
Don't need this file changed, just use a local variable in the one method that needs the notemanager